### ISSUE TITLE
Requests will default to http even if server over https

### ISSUE THREAD

**Issue by ellefsen** [area17/twill#83]
All requests for crud operations in the Twill UI will default to http even if server over https. Are there any environment settings to configure this or to force https? I've tried both with and without scheme in ADMIN_APP_URL and APP_URL, but it doesn't seem to have any effect.

Any suggestions?

**Comment by ifox** [comment]
Hi @ellefsen!

I can't reproduce this, on our end everything is served on https and all requests from CMS we make with Twill are too, without setting up our Laravel apps in a specific way.

@milewski's suggestion would most likely work for you, but I'm curious to hear more about your environment and configuration to figure out why you're experiencing those http requests.

**Comment by ellefsen** [comment]
Hi, thank you for looking into this. It was related to the hosting environment and not Twill itself. Sorry about that.

I had to configure TrustProxies for Heroku to access the X-Forwarded-Proto headers properly. I never had this issue with Heroku and Laravel before, but I realize now that it's because I've mostly been building apis and not relying on url generation in Laravel for that.

For anyone else experiencing this I found two ways to resolve it:

1. Configuring TrustProxies middleware for Heroku
[https://devcenter.heroku.com/articles/getting-started-with-laravel](https://devcenter.heroku.com/articles/getting-started-with-laravel)

2. Setting `HTTPS=on` as an environment variable
